deployment-server/app/Main.hs

```diff
@@ -1,23 +1,23 @@
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
Is vertical alignment of pragmas often a problem? They look so ugly when unaligned...

Reply:
I'd also be happy with always using the same amount of spaces before `#-}` to make each pragma fit. I.e. align pragmas on the project level, not the file level.